src/shared/utils/getEntityWithoutProperties.ts

```diff
@@ -0,0 +1,9 @@
+export const getEntityWithoutProperties = (data: any, keys: string[]): any => {
```

Comment:
insteand of any here would be used as Record<string, any> and return should be Record<string, any> and you could use data.hasOwnProperty instead of calling prototype hasOwnProperty